RecoHGCal/TICL/interface/ClusterFilterByAlgo.h

```diff
@@ -0,0 +1,35 @@
+// Author: Marco Rovere - marco.rovere@cern.ch
+// Date: 11/2018
+// Copyright CERN
+
+#ifndef RecoHGCal_TICL_ClusterFilterByAlgo_H__
+#define RecoHGCal_TICL_ClusterFilterByAlgo_H__
```

Comment:
http://cms-sw.github.io/cms_coding_rules.html#plugins
apparently this and other filters which will be available only via a factory should be moved to the /plugins dir